source/content/environment-specific-config.md

```diff
@@ -11,25 +11,35 @@ In order to improve the development and debugging processes, you might use setti
 
 This doc shows how to use the same codebase with different settings for each environment, using values for the [PANTHEON_ENVIRONMENT variable](/read-environment-config). To quickly see which environment you are on, consider installing the [Pantheon HUD plugin](https://wordpress.org/plugins/pantheon-hud/).
 
-## Use the WP_ENVIRONMENT_TYPE Function to Perform Actions Based on Environment
+## Define WP_DEBUG to Perform Actions Based on Environment
 
-<Partial file="wp_get_environment_type.md" />
-
-### Configure WP_DEBUG in wp-config.php
-
-To conditionally set `WP_DEBUG` based on the given Pantheon environment, change the [existing code block](https://github.com/pantheon-systems/WordPress/blob/default/wp-config.php#L72-L74) defining it:
+To conditionally set `WP_DEBUG` based on the given Pantheon environment, change the [existing code block](https://github.com/pantheon-systems/WordPress/blob/default/wp-config.php#L72-L74) defining it. This example configuration enables `WP_DEBUG` for development environments (Dev and Multidevs), while disabling it on production environments (Test and Live):
 
 ```php:title=wp-config.php
-if ( ! defined( 'WP_DEBUG' ) ) {
-    define('WP_DEBUG', (wp_get_environment_type() == "development"));
+/**
+ * WordPress debugging mode.
+ *
+ * Sets WP_DEBUG to true on if on a development environment.
+ *
+ */
+if (!defined('WP_DEBUG') && isset($_ENV['PANTHEON_ENVIRONMENT'])) {
+    if(in_array( $_ENV['PANTHEON_ENVIRONMENT'], array('test', 'live'))) {
+      define('WP_DEBUG', false);
+    } else {
+      define( 'WP_DEBUG', true );
```

Comment:
You can improve the original example by replacing the if / else with a single statement:
```
define( 'WP_DEBUG', !in_array( $_ENV['PANTHEON_ENVIRONMENT'], array('test', 'live')) );
```
Or, better yet, if our examples do not need to support PHP 5.3 any longer:
```
define( 'WP_DEBUG', !in_array( $_ENV['PANTHEON_ENVIRONMENT'], ['test', 'live']) );
```

Reply:
updated to  read:
```
if (!defined('WP_DEBUG') && isset($_ENV['PANTHEON_ENVIRONMENT'])) {
    define( 'WP_DEBUG', !in_array( $_ENV['PANTHEON_ENVIRONMENT'], ['test', 'live']) );
}
```